build.py

```
@@ -1,6 +1,7 @@
 import PyInstaller.__main__
 import os
 import sys
+import shutil
```

Ruff (0.8.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)